contrib/completion/zsh/_docker

```diff
@@ -253,6 +253,11 @@ __docker_network_subcommand() {
             _arguments -A '-*' \
                 $opts_help \
                 "($help -d --driver)"{-d,--driver=}"[Driver to manage the Network]:driver:(null host bridge overlay)" \
+                "($help)--ipam-driver=-[IP Address Management Driver]:driver:(default)" \
+                "($help)*--subnet=-[Subnet in CIDR format that represents a network segment]:IP/mask: " \
+                "($help)*--ip-range=-[Allocate container ip from a sub-range]:IP/mask: " \
+                "($help)*--gateway=-[ipv4 or ipv6 Gateway for the master subnet]:IP: " \
+                "($help)*--aux-address[Auxiliary ipv4 or ipv6 addresses used by network driver]:key=IP: " \
```

Comment:
@sdurrheimer There is an additional option `--opt|-o`, see https://github.com/docker/docker/pull/17046/files#diff-37d5f4118ca6b64624af89485f9a0f61R49